app/src/main/kotlin/dev/msfjarvis/aps/ui/firstrun/fragments/CloneFragment.kt

```diff
@@ -1,32 +1,47 @@
 /*
  * Copyright © 2019-2020 The Android Password Store Authors. All Rights Reserved.
- * SPDX-License-Identifier: GPL-3.0-only
+ *  SPDX-License-Identifier: GPL-3.0-only
+ *
  */
-package dev.msfjarvis.aps.ui.fragments
+package dev.msfjarvis.aps.ui.firstrun.fragments
 
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
 import dev.msfjarvis.aps.databinding.FragmentCloneBinding
+import dev.msfjarvis.aps.di.activityViewModel
+import dev.msfjarvis.aps.di.injector
 import dev.msfjarvis.aps.utils.performTransactionWithBackStack
 
 class CloneFragment : Fragment() {
 
-  private lateinit var binding: FragmentCloneBinding
+  private var _binding: FragmentCloneBinding? = null
+  private val binding get() = _binding!!
```

Comment:
This adds the overhead of runtime checks on every binding invocation, just stick to the `lateinit var` pattern. I don't trust the Android docs on this.

Reply:
Yeah, i just saw it in a google repo so went with this. Will revert.